raft-tech/TANF-app
# 1113.0 - Chore/reorganize search indexes

## Summary of Changes
As part of 1113, we need to add more models to `search_indexes` representing the SSP and TRIBAL submission. This change restructures the existing `search_indexes` to use folders for models, documents, and admin definitions.

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && npm run start
cd tdrs-backend && docker-compose up
```

1. Open http://localhost:3000/ and sign in.
1. Upload a Section 1 DataFile.
1. Verify T1 data is saved as expected, elastic search is updated.
   * here, it was necessary to run `python manage.py search_index --rebuild` after the backend was started, both locally and in cloud.gov. in cloud.gov this can only be validated via ssh-ing into the backend application and querying elastic via the python shell
   
   
<img width="945" alt="image" src="https://user-images.githubusercontent.com/111305129/230939665-3c92efbc-2009-4b6b-80d4-6c28c606532e.png">


## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
